app/models/badges/v3_pioneer_badge.rb

```diff
@@ -0,0 +1,186 @@
+module Badges
+  class V3PioneerBadge < Badge
+    seed "V3 Pioneer",
+      :ultimate,
+      'v3-pioneer',
+      'Awarded for having helped build Exercism version 3'
+
+    def award_to?(user)
+      V3_PIONEERS.include?(user.handle.downcase)
```

Comment:
The downcasing is to prevent not awarding the badge for handles that are different from the original casing as specified in the Slack thread

Reply:
Are the handles exercism handles or GH handles?